mep/people/models.py

```diff
@@ -306,6 +307,14 @@ def save(self, *args, **kwargs):
 
         super(Person, self).save(*args, **kwargs)
 
+    def get_absolute_url(self):
+        '''
+        Return the public url to view library member's detail page
```

Comment:
do we care about triple double quotes/same line for docstrings? thought I remembered discussion about it on parasol

Reply:
good catch, thanks for the reminder - it will probably take me a while to remember that and shift my practice